scratch/Dataset.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
Is dit notebook nodig? Wat doet het precies?

Reply:
Ik dacht dat ik deze eruit had gehaald maar blijkbaar niet :)